### ISSUE TITLE
I cant find sscanf, how do I add it?

### ISSUE THREAD

**Issue by hexbinoct** [Jonimoose/libfxcg#79]
sscanf is not in stdio.h, can someone please guide me how do I add it so I can use it in the examples code?

**Comment by dr-carlos** [comment]
I don't think that libfxcg currently includes sscanf (a simple search of the repo shows this).
You will have to implement it yourself. If you would then like to, you could probably make a PR with it in libc/stdio.c.

**Comment by hexbinoct** [comment]
Yes i had to do it myself, but I didnt push back the changes, is it worth doing it, no one replies here anyway?

**Comment by dr-carlos** [comment]
You don't have to; it's not very active, but usually PRs do get looked at.
Anyway, feel free to close the issue if you managed to fix it.
